DOCS-1831: Update deploy to mention where to find all models (#2538)
viamrobotics/docs: docs/ml/deploy.md

## Create an ML model service

> You can use the ML model service to deploy:

Author: ```suggestion
You can use the ML model service to deploy:

```

> The Viam registry hosts trained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your robot instead of training your own.

Reviewer: ```suggestion
The Viam registry hosts trained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your machine instead of training your own.
```

You can see all publicly available ML models as well as all ML models you've uploaded yourself in the [Viam Registry](https://app.viam.com/registry).

> Once you've decided on a model to use, copy the name of the ML model.

Reviewer: I would remove this line. Here we advise that the user can copy the name of a chosen ML model, but all future references to this ML model selection explicitly specify the built-in only: `TFLite CPU`. If we do want to retain "you can pick any", following copy should likely also specify any rather than specific. Example for line 63:

> Select the `ML Model` type, then select the ML model you would like to use, such as the built-in `TFLite CPU` model.

Author: it's the ml model not the model name so "effdet0" for example...

Author: oh I see that doesn't make sense, agreed
```suggestion
```

Author: maybe

> The Viam registry hosts trained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your machine instead of training your own.
> You can see all publicly available ML models as well as all ML models you've uploaded yourself in the [Viam Registry](https://app.viam.com/registry).
> 
> Once you've decided on a model to use, copy the name of the ML model.

Author: ```suggestion
You can use the ML model service to deploy:
- [an existing model from the registry](#select-an-existing-model-from-the-registry)
- a model trained outside the Viam platform that you have [uploaded](#upload-an-existing-ml-model)
- a model available on your machine
```

## Create an ML model service

You can use the ML model service to deploy:

> - [an existing model from the registry](#select-an-existing-model-from-the-registry)

Author: ```suggestion
- [an existing model from the registry](https://app.viam.com/registry)
```

> - a model trained outside the Viam platform that you have [uploaded](#upload-an-existing-ml-model)

Author: ```suggestion
- a model trained outside the Viam platform that you have [uploaded](/ml/upload-model/)
```
